libs/python.json

```diff
@@ -118,6 +118,8 @@
   {"id":"py.scrapy","imports":["scrapy"],"name":"scrapy","repo":"scrapy/scrapy","tags":["web-crawler"],"tech":["internet"],"status":"model-ready"},
   {"id":"py.urllib3","imports":["urllib3"],"name":"urllib3","repo":"urllib3/urllib3","tags":["http-client"],"tech":["internet"],"status":"model-ready"},
   {"id":"py.six","imports":["six"],"name":"Six","repo":"benjaminp/six","tags":["python2-python3"],"tech":["interoperability"],"status":"model-ready"},
+  {"id":"py.RPi-GPIO","imports":["RPi.GPIO"],"name":"RPi.GPIO","repo":"","tags":["automation"],"tech":["iot"],"examples":["https://github.com/nosix/raspberry-gpio-emulator"],"status":"awaiting-model"},
```

Comment:
No repo? Is this an omission?

Reply:
No  @sergey48k  it's not an omission the repo is not available on GitHub